src/Tools/dotnet-monitor/Egress/AzureBlob/AzureBlobEgressProvider.cs

```diff
@@ -134,6 +139,10 @@ public AzureBlobEgressProvider(ILogger<AzureBlobEgressProvider> logger)
             {
                 throw CreateException(ex);
             }
+            catch (CredentialUnavailableException ex)
+            {
+                throw CreateException(ex);
```

Comment:
Can you provide an example of the egress operation status when the credentials are invalid (e.g. use a random client ID)?

Reply:
``` json
{"resourceLocation":null,"error":{"code":"400","message":"Azure blob egress failed: ManagedIdentityCredential authentication unavailable. The requested identity has not been assigned to this resource.\nStatus: 400 (Bad Request)\n\nContent:\n{\"error\":\"invalid_request\",\"error_description\":\"Identity not found\"}\n\nHeaders:\nServer: IMDS/150.870.65.597\nDate: Sat, 21 May 2022 05:18:04 GMT\nContent-Type: application/json; charset=utf-8\nContent-Length: 68\n"},"operationId":"904d419c-a45b-4cd4-b8a3-5debaa7fbd6a","createdDateTime":"2022-05-21T05:18:03.9082054Z","status":"Failed"}
```